src/docs/metrics/community-messages-count/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Community Messages Count
+author: Ivan
+date: 2020-04-13
+# REF metrics-hub/metricshub/community_messages.py
+---
+
+## Definition
+
+Community Messages Count is build on top of the [Social Data](/metrics/details/social-data).
+
+Community Messages Count for an asset compares the [Social
+Volume](/metrics/social-volume) returns the number of messages written in the
+asset's own media server.
```

Comment:
@IvanIvanoff @ipranovich is it a good wordings? "Media server"

Reply:
@IvanIvanoff  @Partysun 
I'm not sure about media server, tbh... Maybe community channel/server. Maybe even better to ask Dino or Kim about proper general name for such kind of things?